.github/workflows/build.yml

```
@@ -108,7 +108,7 @@ jobs:
           cargo build
           popd
           pushd ffi
-          cargo b --features default-engine,sync-engine
+          cargo b --features default-engine,sync-engine,test_ffi
```

nit for consistency:
```suggestion
          cargo b --features default-engine,sync-engine,test-ffi
```